tests/test_google_ads_interrupted_sync_remove_currently_syncing.py

```diff
@@ -0,0 +1,264 @@
+         - Verify an interrupted sync can resume based on the `currently_syncing` and stream level bookmark value
+         - Verify only records with replication-key values greater than or equal to the stream level bookmark are replicated on the resuming sync for the interrupted stream
+         - Verify the yet-to-be-synced streams are replicated following the interrupted stream in the resuming sync. All yet-to-be-synced streams must replicate before streams that were already synced.
+        """
+        print("Interrupted Sync Test for tap-google-ads with added stream")
+
+        # the following streams are under test as they all have 4 consecutive days with records e.g.
+        # ('2022-01-23T00:00:00.000000Z', '2022-01-23T00:00:00.000000Z', '2022-01-24T00:00:00.000000Z', '2022-01-25T00:00:00.000000Z')])}
+        streams_under_test = {'account_performance_report',
+                              'search_query_performance_report',
+                              'user_location_performance_report',
+        }
+
+        # Create connection using a recent start date
+        self.start_date = '2022-01-22T00:00:00Z'
+        conn_id = connections.ensure_connection(self, original_properties=False)
+
+        # Run a discovery job
+        found_catalogs_1 = self.run_and_verify_check_mode(conn_id)
+
+        # partition catalogs for use in table/field seelction
+        test_catalogs_1 = [catalog for catalog in found_catalogs_1
+                           if catalog.get('stream_name') in streams_under_test]
+        core_catalogs_1 = [catalog for catalog in test_catalogs_1
+                           if not self.is_report(catalog['stream_name'])]
+        report_catalogs_1 = [catalog for catalog in test_catalogs_1
+                             if self.is_report(catalog['stream_name'])]
+
+        # select all fields for core streams
+        self.select_all_streams_and_fields(conn_id, core_catalogs_1, select_all_fields=True)
+
+        # select 'default' fields for report streams
+        self.select_all_streams_and_default_fields(conn_id, report_catalogs_1)
+
+        # Run a sync
+        full_sync = self.run_and_verify_sync(conn_id)
+
+        # acquire records from target output
+        full_sync_records = runner.get_records_from_target_output()
+        full_sync_state = menagerie.get_state(conn_id)
+
+        # Add a stream between syncs
+        removed_stream = 'search_query_performance_report'
+        streams_under_test.remove(removed_stream)
+
+        test_catalogs_1 = [catalog for catalog in found_catalogs_1
+                           if catalog.get('stream_name') in streams_under_test]
+
+        report_catalogs_1 = [catalog for catalog in test_catalogs_1
+                             if self.is_report(catalog['stream_name'])]
+
+        # select 'default' fields for report streams
+        self.select_all_streams_and_default_fields(conn_id, report_catalogs_1)
```

Comment:
Will need to use the deselect for just the one stream here, instead of doing the selection.

Reply:
Done.